ERROR - 2025-06-04 00:12:02 --> mysqli_sql_exception: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email' in C:\wamp64\www\proyecto\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\proyecto\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\proyecto\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\proyecto\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\proyecto\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\proyecto\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp64\www\proyecto\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp64\www\proyecto\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\wamp64\www\proyecto\app\Controllers\AuthController.php(73): CodeIgniter\Model->insert(Array)
#8 C:\wamp64\www\proyecto\system\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
#9 C:\wamp64\www\proyecto\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\wamp64\www\proyecto\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\proyecto\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\wamp64\www\proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\wamp64\www\proyecto\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-04 00:12:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
[Method: POST, Route: registro/procesarRegistro]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(73): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:12:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$WUM0edRwHc7xD.hLXVIFSe0WRZeFJ522h.l9zS8vMkQHBtXACvNvG\', \'2025-06-04 00:12:02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$WUM0edRwHc7xD.hLXVIFSe0WRZeFJ522h.l9zS8vMkQHBtXACvNvG\', \'2025-06-04 00:12:02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(73): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:12:02 --> [Caused by] mysqli_sql_exception: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$WUM0edRwHc7xD.hLXVIFSe0WRZeFJ522h.l9zS8vMkQHBtXACvNvG\', \'2025-06-04 00:12:02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$WUM0edRwHc7xD.hLXVIFSe0WRZeFJ522h.l9zS8vMkQHBtXACvNvG\', \'2025-06-04 00:12:02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$WUM0edRwHc7xD.hLXVIFSe0WRZeFJ522h.l9zS8vMkQHBtXACvNvG\', \'2025-06-04 00:12:02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(73): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-04 00:26:12 --> mysqli_sql_exception: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email' in C:\wamp64\www\proyecto\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\wamp64\www\proyecto\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\wamp64\www\proyecto\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\wamp64\www\proyecto\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\wamp64\www\proyecto\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\wamp64\www\proyecto\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp64\www\proyecto\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp64\www\proyecto\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\wamp64\www\proyecto\app\Controllers\AuthController.php(76): CodeIgniter\Model->insert(Array)
#8 C:\wamp64\www\proyecto\system\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
#9 C:\wamp64\www\proyecto\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\wamp64\www\proyecto\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\proyecto\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\wamp64\www\proyecto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\wamp64\www\proyecto\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-04 00:26:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
[Method: POST, Route: registro/procesarRegistro]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(76): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:26:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maiximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$6JkXjYYc4vw/WjDprsMVm.jBbCe5KyDerZMp.BROEK1BUleXFuuCm\', \'2025-06-04 00:26:12\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maiximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$6JkXjYYc4vw/WjDprsMVm.jBbCe5KyDerZMp.BROEK1BUleXFuuCm\', \'2025-06-04 00:26:12\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(76): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:26:12 --> [Caused by] mysqli_sql_exception: Duplicate entry 'maxis.c.22@gmail.com' for key 'usuarios.email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maiximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$6JkXjYYc4vw/WjDprsMVm.jBbCe5KyDerZMp.BROEK1BUleXFuuCm\', \'2025-06-04 00:26:12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maiximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$6JkXjYYc4vw/WjDprsMVm.jBbCe5KyDerZMp.BROEK1BUleXFuuCm\', \'2025-06-04 00:26:12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (\'Maiximiliano\', \'Correa\', \'maxis.c.22@gmail.com\', \'$2y$10$6JkXjYYc4vw/WjDprsMVm.jBbCe5KyDerZMp.BROEK1BUleXFuuCm\', \'2025-06-04 00:26:12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `password`, `creado_en`) VALUES (:nombre:, :apellido:, :email:, :password:, :creado_en:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(76): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registrar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:49:14 --> ErrorException: Array to string conversion
[Method: GET, Route: registro]
in APPPATH\Views\registro.php on line 18.
 1 APPPATH\Views\registro.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\wamp64\\www\\proyecto\\app\\Views\\registro.php', 18)
 2 SYSTEMPATH\View\View.php(224): include('C:\\wamp64\\www\\proyecto\\app\\Views\\registro.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('registro', [], true)
 5 APPPATH\Controllers\AuthController.php(40): view('registro')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registro()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-04 00:53:04 --> ErrorException: Array to string conversion
[Method: GET, Route: registro]
in APPPATH\Views\registro.php on line 18.
 1 APPPATH\Views\registro.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\wamp64\\www\\proyecto\\app\\Views\\registro.php', 18)
 2 SYSTEMPATH\View\View.php(224): include('C:\\wamp64\\www\\proyecto\\app\\Views\\registro.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('registro', [], true)
 5 APPPATH\Controllers\AuthController.php(40): view('registro')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registro()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
